rpm/generic/zfs.spec.in

```diff
@@ -179,11 +179,12 @@ Requires:       lsscsi
 Requires:       mdadm
 Requires:       bc
 Requires:       ksh
-Requires:	fio
-Requires:	acl
-Requires:	sudo
-Requires:	sysstat
+Requires:       fio
+Requires:       acl
+Requires:       sudo
+Requires:       sysstat
 Requires:       rng-tools
+Requires:       bzip2
```

Comment:
The ZTS requires a lot more (https://github.com/zfsonlinux/zfs/blob/master/tests/zfs-tests/include/commands.cfg) than `bzip2` to be run, i don't think we can/should add every dependency in the rpm specfile.

The test runner warns about missing utilities on startup anyway:

```
--- Configuration ---
Runfile:         /usr/share/zfs/runfiles/linux.run
STF_TOOLS:       /usr/share/zfs/test-runner
STF_SUITE:       /usr/share/zfs/zfs-tests
STF_PATH:        /var/tmp/constrained_path.5u2L
FILEDIR:         /var/tmp
FILES:           /var/tmp/file-vdev0 /var/tmp/file-vdev1 /var/tmp/file-vdev2
LOOPBACKS:       /dev/loop5 /dev/loop6 /dev/loop7 
DISKS:           loop5 loop6 loop7 
NUM_DISKS:       3
FILESIZE:        4G
Keep pool(s):    testpool
Missing util(s): bzcat exportfs fio net pax quotaon setenforce umask uuidgen wait 
```

Is `bzip2/bzcat` really **required** by the ZTS? If it is we should probably also update `zfs-buildbot/master/scripts/bb-dependencies.sh`.

Reply:
Since `zfs-tests` is a sub-package which doesn't get installed by default, it seems reasonable to at least look in to adding all the missing dependencies to the spec file.  I expect that most of the utilities the ZTS depends on are already part a a minimum install anyway.